helix-term/src/ui/editor.rs

```diff
@@ -259,11 +261,15 @@ impl EditorView {
                 // Standard case.
                 let cursor_start = prev_grapheme_boundary(text, range.head);
                 spans.push((selection_scope, range.anchor..cursor_start));
-                spans.push((cursor_scope, cursor_start..range.head));
+                if i != primary_idx {
+                    spans.push((cursor_scope, cursor_start..range.head));
+                }
```

Comment:
Previously the terminal cursor was hidden and we drew our own primary cursor. Now we use the terminal cursor as the primary cursor and hence `ui.cursor.primary` is no longer used to style it. Is there any way to use it to style the cursor ?

Reply:
Can you please add a comment for these?

Reply:
I was looking for a way to style the terminal cursor so that we can still use `ui.cursor.primary`; it's possible to do so and vim allows it: https://github.com/helix-editor/helix/issues/323#issuecomment-866346854.